### ISSUE TITLE
Change wordQueue from an array to an object

### ISSUE THREAD

**Issue by AidanWelch** [AidanWelch/ProgressiveImmersion#14]
For this amount of data its not that important to save on property names, and it just makes the code more confusing

**Comment by JoshLamptey** [comment]
can you please try to explain the problem more simply for someone who didn't work on the codebase to understand cause I've spent the past 45 minutes looking for any variable named "wordQueue"

**Comment by AidanWelch** [comment]
@JoshLamptey Absolutely!  Sorry for my bad explanation of the problem before.  Basically, `content_scripts/analyze.js` takes has a tally of words that meet our requirements for good words to translate.  When the a page is closed it tries to save `wordsToSave`(default 5) number of words to the `browser.storage.local.get('wordQueue')`.  The wordQueue is an array of up to 100 elements.  Each element is an array itself with 3 elements, index 0 is the word itself, index 1 is the number of times the word has occurred in webpages we analyzed, and index 2 is the epoch time it last occurred.  The wordQueue is automatically popped from and translated in `background_scripts/dictionary-handler.js`.  So, basically since every element in wordQueue is always a length 3 array, might as well just make each element an object instead and then they can have more clear names.

**Comment by JoshLamptey** [comment]
Okay I'm at work when I get home I'll get on this. Thank you
